spid_cie_oidc/authority/views.py

```diff
@@ -68,6 +76,42 @@ def entity_list(request):
     return JsonResponse(list(set(entries)), safe=False)
 
 
+def advanced_entity_listing(request):
+    desecendants = FederationDescendant.objects.filter(
+        is_active = True,
+    ).order_by("modified")
+    entities_list = []
+    for descendant in desecendants:
+        entity = {
+            descendant.sub : {
+                "iat" : int(descendant.modified.timestamp())
+            }
+        }
+        entities_list.append(entity)
+    total_entries = desecendants.count()
+    max_entry_page = getattr(settings, "MAX_ENTRIES_PAGE", 100)
+    p = Paginator(entities_list, max_entry_page)
+    page = request.GET.get("page", 1)
+    entities = p.get_page(page)
+    next_page_path = ""
+    if entities.has_next():
+        next_page_path = f"advanced_entity_listing?page={entities.next_page_number()}"
+    prev_page_path = ""
+    if entities.has_previous():
+        prev_page_path = f"advanced_entity_listing?page={entities.previous_page_number()}"
+    res = {
+            "iss" : get_first_self_trust_anchor().sub,
```

Comment:
is the trust anchor is missing this code would produce an error 500

please handle the exception in the right way